src/main.cpp

```diff
@@ -669,51 +691,57 @@ int main(int argc, char* argv[])
 		return 0;
 	}
 
-	if (cli.command == "version" || cli.command == "--version" || cli.command == "-version" || cli.command == "-v") {
+	if (cli.command == "version" || cli.command == "--version" || cli.command == "-version") {
 		logf(g_version_string);
 		return 0;
 	}
 
-	if (argc == 2) {
-		start_viewer(argv[1]);
+	if (cli.command == "editor" || cli.command == "empty") {
+		start_viewer(std::string());
+		return 0;
 	}
-	else
-	{
-		if (cli.bspfile.empty()) {
-			logf("ERROR: no map specified\n"); 
-			return 1;
-		}
 
-		if (cli.hasOption("-v")) {
-			g_verbose = true;
-		}
+	if (cli.bspfile.empty()) {
+		logf("ERROR: no map specified\n"); 
+		return 1;
+	}
 
-		if (cli.command == "info") {
-			return print_info(cli);
-		}
-		else if (cli.command == "noclip") {
-			return noclip(cli);
-		}
-		else if (cli.command == "simplify") {
-			return simplify(cli);
-		}
-		else if (cli.command == "delete") {
-			return deleteCmd(cli);
-		}
-		else if (cli.command == "transform") {
-			return transform(cli);
-		}
-		else if (cli.command == "merge") {
-			return merge_maps(cli);
-		}
-		else if (cli.command == "unembed") {
-			return unembed(cli);
-		}
-		else {
-			logf("unrecognized command: %d\n", cli.command.c_str());
-		}
+	if (cli.command == "exportobj") {
+		Bsp* tmpBsp = new Bsp(cli.bspfile);
+		tmpBsp->ExportToObjWIP(cli.bspfile);
+		delete tmpBsp;
+		return 0;
+	}

```

Comment:
this can move down with the other commands

Reply:
Move down? How i can understand it ?